bot/cogs/cyber.py

```diff
@@ -90,6 +90,7 @@ def __init__(self, bot: Bot):
             challenge_title = challenge_raw["title"]
             challenge_difficulty = challenge_raw["difficulty"]
             challenge_text = challenge_raw["description"]
+            challenge_text += f"\nhttps://assess.joincyberdiscovery.com/challenge-{challenge_num:02d}"
```

Comment:
Great idea! Could it be more user friendly to utilise the URL property of embeds, making them easily clickable, without the need for a full link URL to be visible in the description text area?

https://discordpy.readthedocs.io/en/rewrite/api.html#discord.Embed.url

Reply:
Alternatively, embeds support better markdown

Reply:
Alternatively, embeds support better markdown

Reply:
Alternatively, embeds support better markdown